src/main/webapp/pages/common/banner.jsp

```diff
@@ -92,18 +94,33 @@ function getCsrfToken() {
  		<div style="display: block">
 			<%
 				UserSessionData usd = null;
+				NotificationService ns = new NotificationService(); 
```

Comment:
This works fine but not a good pattern .
the reason we use MVC pattern is for separation of concerns